driver-core/src/main/com/mongodb/connection/SocketStreamFactory.java

```diff
@@ -85,7 +87,7 @@ public Stream create(final ServerAddress serverAddress) {
 
     private SSLContext getSslContext() {
         try {
-            return (sslSettings.getContext() == null) ? SSLContext.getDefault() : sslSettings.getContext();
+            return ofNullable(sslSettings.getContext()).orElse(SSLContext.getDefault());
```

Comment:
Comment: For some reason I find using `Optional.ofNullable` easier to grok :shrug: